components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/constant/SQLConstants.java

```diff
@@ -165,6 +238,66 @@
             "  R.NAME = ?\n" +
             "  AND R.TENANT_ID = ?\n" +
             "  AND R.TYPE_ID = ?";
+    public static final String GET_RESOURCE_BY_NAME_MSSQL = "SELECT\n" +
+            "  R.ID,\n" +
```

Comment:
let's avoid adding the \n.. new line character is not required in java files.

Reply:
addressed in ff47c030b9c14da99a0a8781ef12c24ffd127ff3